change/@uifabric-azure-themes-2019-09-02-17-00-24-master.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "Fix dropdownItem style override for azure-themes.",
```

Comment:
Please change to "DropDown: Fixed dropdownItem and dropdownItemSelected azure-theme styles"